crates/swc_ecma_transforms_proposal/src/decorators/legacy/metadata.rs

```diff
@@ -428,7 +424,7 @@ fn serialize_type(class_name: Option<&Ident>, param: Option<&TsTypeAnn>) -> Expr
                     kind: TsKeywordTypeKind::TsUndefinedKeyword,
                     ..
                 }) => {
-                    continue;
+                    return quote_ident!("Object").into();
```

Comment:
`string | null` => `Object` and `string | undefined` => `Object` according to TS

https://www.typescriptlang.org/play?#code/MYewdgzgLgBAKgCwJZgOYBECmoBOBDKEHALhgAUcQAHTHKATy1wKJgF4YAKASnYD4YAbwC+AWABQEmDGAAbPBAjxkaIVOkwAAohQZsRFjnXSoK1Jyp58AWwCMpaDl0wAPjDABXWbIA0MSzYATA5QTqpuHmAAJpgAZiiYUX68IurCQA